packages/fether-electron/babel.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  plugins: ['@babel/plugin-proposal-class-properties'],
```

Comment:
Let's add `'@babel/preset-stage-0'` as in fether-ui

Reply:
I've updated to higher than Babel 7 beta, removed stage-0 and added individual plugins for Babel 7 according to the migration guide